automated/linux/basic-tests/basic-test-0001_repo.yaml

```diff
@@ -0,0 +1,20 @@
+metadata:
+    name: Basic test 0001 - Repo
+    format: "Lava-Test Test Definition 1.0"
+    description: "This test checks if RevolutionPi-Repository works."
+    maintainer:
+        - r.gsponer@kunbus.com
+    os:
+        - raspian
+
+    scope:
+        - functional
+    devices:
+        - Core3_Ostfildern_001
+
+run:
+    steps:
+        - lava-test-case test1 --result pass
```

Comment:
Wo/was ist die "test1" ?

Reply:
That is just a name... It can be called differently.

Reply:
so, this means no responding testcase in test script defined.
A suggestion would be,  maybe make test case with python, a testcase with shell, and perl, etc.  Both to test the environment  also make examples for every kind of script.